packages/remix-react/scroll-restoration.tsx

```diff
@@ -1,49 +1,62 @@
 import * as React from "react";
-import { useLocation } from "react-router-dom";
+import type { ScrollRestorationProps as ScrollRestorationPropsRR } from "react-router-dom";
+import {
+  useLocation,
+  UNSAFE_useScrollRestoration as useScrollRestoration,
```

Comment:
The hook was private in both Remix and RR so kept it that way via the `UNSAFE_` prefix